app/components/ApplicationPage/AuthPage/SSOCallbackPage/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
```

Comment:
I'm assuming users are sent here from the backend after the external signin succeeds? Can you add some comments about how users end up on this page, since it falls outside of the typical routing system and is not immediately clear?